src/components/carousel/index.tsx

```diff
@@ -466,8 +476,14 @@ class Carousel extends Component<CarouselProps, CarouselState> {
     const marginBottom = Math.max(0, this.getContainerPaddingVertical() - 16);
     const ScrollContainer = animatedScrollOffset ? Animated.ScrollView : ScrollView;
     return (
-      <View animated={animated} style={[{marginBottom}, containerStyle]} onLayout={this.onContainerLayout}>
+      <View
+        animated={animated}
+        style={[{marginBottom}, containerStyle]}
+        onLayout={this.onContainerLayout}
+        testID={testID}
```

Comment:
As we talked about in the meeting, we usually pass the testID to the main component and not to the wrapping view. consider passing the testID directly to the scrollView.

Reply:
If I'll do that I won't be able to test anything on the container. I don't think it's the right call in this case...

Reply:
What do you want to test on the container? I saw you don't have any test for it at the moment..

Reply:
You know what? Since {...others} is passed to the ScrollView, I'll leave the testID there so I won't break anyone. Thanks